draft/spec/index.html

```diff
@@ -383,6 +385,12 @@ <h2>Version Directories</h2>
             Empty directories within a version directory are not permitted. Otherwise empty directories MAY be
             maintained by creating a <code>.keep</code> file within that directory.
         </p>
+        <p>
+            Version directories MUST have a sub-directory called <code>content</code> if there are files present, and
+            SHOULD NOT contain a <code>content</code> subdirectory otherwise. There MUST be no other files or
```

Comment:
The other instances of "subdirectory" are spelled with a hyphen: "sub-directory".

Reply:
fixed